docs/DEPLOYMENT.md

```diff
@@ -150,16 +181,12 @@ metadata:
   labels:
     app: api
 data:
-  CONFIG_FILE_URL: https://raw.githubusercontent.com/tektoncd/hub/master/config.yaml        ## Change the file URL here
-
+  CONFIG_FILE_URL: https://raw.githubusercontent.com/tektoncd/hub/master/config.yaml ## Change the file URL here
 ```
 
-In this file we add users who have additional scopes which can be used to
-- refresh a catalog,
-- refresh config file 
-- create an agent token 
+All users have default scopes `rating:read` and `rating:write` while they get the other scopes after login which allows them to rate the resources.
```

Comment:
`while they get the other scopes after login which allows them to rate the resources.`
may be reframe this